README.md

```diff
@@ -88,6 +89,32 @@ To disable `url()` resolving by `css-loader` set the option to `false`
 }
 ```
 
+#### `{RegExp}`
```

Comment:
A few words about what these options do would be smashing!

Reply:
yeah yeah :) I intend to appropriate documentation before this goes anywhere, it hasn't even been signed-off by other maintainers nor released yet... 😛

Reply:
Oh man, was already hoping to use it today 😆 😭